packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+- [PasswordInput](#passwordinput)
+- [Notes](#notes)
+- [Open questions](#open-questions)
+- [References](#references)
+
+## Design
+
+- [TextInput](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=10953%3A191574)
+- [PasswordInput](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=10953%3A180381)
+- [OTPInput]()
+- [CardInput]()
+- [TextArea]()
+
+## BaseInputField API
+
+## TextInput API
+
+**Sample Usage**
+
+```tsx
+<TextInput
+  label="Enter Name"
+  labelPosition="left"
+  placeholder="Enter your name"
+  leadingIcon={UserIcon}
+  showClearButton
+  helpText="Enter first name and last name. Eg: Kamlesh Chandnani"
+  errorText="Name cannot be blank"
+  successText="Name validated"
+  validationState={inputValue.length < 10 ? 'error' : 'success'}
+/>
+```
+
+The mobile keyboard type will be automatically determined based on the `type` attribute. Check out [Web & React Native Attributes for showing keyboard types and autocomplete suggestions based on input](#web--react-native-attributes-for-showing-keyboard-types-and-autocomplete-suggestions-based-on-input) section for more details
+
+| Prop                                                                                               | Type                                                                      | Required                                                                            | Default                                                   | Description                                                                                                                                                                                                                                                                           |
+| -------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- | ----------------------------------------------------------------------------------- | --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type                                                                                               | `text`, `search`, `tel`, `email`, `url`, `numeric`                        | No                                                                                  | `text`                                                    | type of text to be rendered. this will also determine the keyboard type to be show on mobile devices                                                                                                                                                                                  |
```

Comment:
What should we do for `numeric` input? 

Do we show those up/down spinner icons? Do we have designs for this?

Reply:
check the table where I've mentioned the input type and their corresponding input mode. this case is covered there. we don't show those up/down arrow